console/src/components/Common/utils/pgUtils.js

```diff
@@ -77,6 +77,17 @@ export const getTrackedTables = tables => {
   return tables.filter(t => t.is_table_tracked);
 };
 
+export const isColumnAutoIncrement = column => {
```

Comment:
We sure this is the foolproof way of checking autoIncrement?
Because the sequence could refer to a custom sequence too. We should be wary of that since we support existing databases.

Something like this would be safer? What do you think? Its a false-negative vs false-positive problem.

```js
column.column_default && column.column_default.includes("_seq'::regclass)");
```